tests/integration/kubernetes/gha-run.sh

```diff
@@ -284,6 +284,8 @@ function deploy_nydus_snapshotter() {
 	git clone -b "${nydus_snapshotter_version}" "${nydus_snapshotter_url}" "${nydus_snapshotter_install_dir}"
 
 	pushd "$nydus_snapshotter_install_dir"
+	kubectl delete -f "misc/snapshotter/nydus-snapshotter-rbac.yaml" || true
```

Comment:
@ChengyuZhu6, please, instead of just deleting the RBAC here, I'd strongly suggest you call the cleaup method so we make sure that the daemon-set is also deleted.  Deleting just the RBAC doesn't solve the issue.

Reply:
@ChengyuZhu6, on a local cluster I noticed something quite interesting, if the namespace is empty, every now and then it'd be in a "Terminatting" state forever, not actually deleting it, neither being able to deploy the nydus-snapshotter without an error (as the namespace already exists at the deployment time).

I "solved" this by doing:
```sh
kubectl get ns nydus-system -o json | jq '.spec.finalizers = []' | kubectl replace --raw "/api/v1/namespaces/nydus-system/finalize" -f 
```

So, I think this is worth adding to ensure it's actually gone.

Reply:
Fixed. Thanks for your help @fidencio !